web/migrations/0067_virtualclassroomwhiteboard.py

```
@@ -0,0 +1,31 @@
+    dependencies = [
+        ('web', '0066_virtualclassroomparticipant_last_active'),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='VirtualClassroomWhiteboard',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('canvas_data', models.JSONField(default=dict, help_text='Canvas drawing data in JSON format')),
+                ('background_image', models.TextField(blank=True, help_text='Base64 encoded background image', null=True)),
```

Pylint (3.3.7)

[convention] Line too long (123/100)

(C0301)